pkg/sriov/sriov.go

```diff
@@ -247,6 +265,33 @@ func (s *sriovManager) ApplyVFConfig(conf *sriovtypes.NetConf) error {
 	// 2. Set mac address
 
 	// 3. Set link rate
+	if conf.MaxTxRate != nil {
+		if err = s.nLink.LinkSetVfTxRate(pfLink, conf.VFID, *conf.MaxTxRate); err != nil {
+			return fmt.Errorf("failed to set vf %d max_tx_rate to %d Mbps: %v", conf.VFID, conf.MaxTxRate, err)
+		}
+	}
+
+	// 4. Set spoofchk flag
+	if conf.SpoofChk != "" {
```

Comment:
Shall we reset VF config (`spoof check` and `trusted vf`) to original values or default values when VF is released?

Reply:
I had asked the same questions when I first opened this PR, was hoping to get some input from folks.

I think, it would be good to reset to default values when the VF is released. So, in ResetVFConfig(), I could add code to set:

spoofchk -> on
trust -> off
max_tx_rate -> 0, which means no rate limiting.

I don't think it would be required to cache the original value and restore when VF is released, but if others have strong opinion that this is preferred, I could add that as well.

Reply:
I agree, we don't have to cache and restore, resetting to default value shall be fine. 
Just like what we did to vlan, it will be reset to 0.
And for security reason, we should reset spoof check to on and trusted vf to off .

Reply:
OK, let me make that change, test it, and will push an updated patch with those changes.

Reply:
Added another commit that sets the above flags to default values when VF is released.